packages/suite/src/hooks/guide/useGuideLoadArticle.ts

```diff
@@ -3,12 +3,18 @@ import { useEffect, useState } from 'react';
 import type { GuideNode, Locale } from '@suite-common/suite-types';
 
 export const loadPageMarkdownFile = async (id: string, language = 'en-US'): Promise<string> => {
-    const file = await import(
-        /* @vite-ignore */ `@trezor/suite-data/files/guide/${language.toLowerCase()}${id}`
-    );
-    const md = await file.default;
+    const cleanId = id.startsWith('/') ? id : `/${id}`;
+    const url = `/guide/${language.toLowerCase()}${cleanId}`;
 
-    return md;
+    const response = await fetch(url);
+
+    if (!response.ok) {
```

Comment:
The reason of why I didn't make the markdown files working in Vite was not to touch the production code btw. THis is what I wanted to avoid.

Reply:
My immediate guess is that this won't work in the built app, coz the path should be like `file://` so it won't do fetch here.

But I am not sure.

Reply:
Trying a build here: https://github.com/trezor/trezor-suite/actions/runs/18901472081

Reply:
Yeah, I was right unfortunately. In the build, the Guide contents won't show up :(.

Anysways, you work on Guide and you want to use Vite there, right?

Reply:
I want to work for Trezor and just wanted to get a feel of the whole suite codebase and hence picked the issue with `good first issue` label, so a lot of the things are still new for me.